app/src/androidTest/java/org/simple/clinic/drugs/PrescriptionRepositoryAndroidTest.kt

```diff
@@ -202,4 +203,115 @@ class PrescriptionRepositoryAndroidTest {
     )
     assertThat(database.prescriptionDao().getOne(prescribedDrug.uuid)).isEqualTo(expectedPrescribedDrug)
   }
+
+  @Test
+  fun updating_prescribed_drug_teleconsultation_id_should_work_correctly() {
+    // given
+    val durationToAdvanceBy = Duration.ofMinutes(10)
+    val teleconsultationId = UUID.fromString("35756e86-4b31-44cb-bd5f-cd3bc8ad2448")
+    val timestamps = Timestamps(
+        createdAt = Instant.now(clock),
+        updatedAt = Instant.now(clock),
+        deletedAt = null,
+    )
+    val prescribedDrug = TestData.prescription(
+        uuid = UUID.fromString("4ba45cf0-b88c-4efd-8fb0-e46c3b2bb32e"),
+        name = "Amlodipine",
+        frequency = OD,
+        timestamps = timestamps,
+        syncStatus = SyncStatus.DONE,
+        teleconsultationId = null
+    )
+
+    database.prescriptionDao().save(listOf(prescribedDrug))
+
+    // when
+    clock.advanceBy(durationToAdvanceBy)
+    repository.updateTeleconsultationId(
```

Comment:
Why do we need this method which accepts a single drug UUID if we already one that accepts a list?

Reply:
Initially I was trying out a different approach, but left it as a way to just update single teleconsultation id. I can remove it since there is no use case for it currently.